lighthouse-core/report/html/templates.html

```diff
@@ -352,8 +352,32 @@
         margin-left: 0;
       }
     }
+
+    .lh-a11y-quick-focus {
+      background-color: var(--color-blue);
+      color: var(--color-white);
+      padding: 20px;
+      clip: rect(1px, 1px, 1px, 1px);
```

Comment:
why clip?

Reply:
Good question. Here's a bad answer: I took it straight from GitHub.

`display: none -> block` won't work, because you can't focus onto a `display: none` element. `opacity: 0 -> 1` does work, though. I wonder if there might be screen readers that will ignore 0 opacity elements.